src/transformer/fields/adresse/adresse.field.ts

```diff
@@ -1,6 +1,6 @@
-/* eslint-disable @typescript-eslint/naming-convention, camelcase */
+/* eslint-disable @typescript-eslint/naming-convention, camelcase, max-statements, complexity */
```

Comment:
```suggestion
/* eslint-disable @typescript-eslint/naming-convention, camelcase */
```
C'est peut-être possible d'activer ces deux règles en simplifiant un peu le code en introduisant une fonction